lib/widgets/components/entry/register/name_body.dart

```diff
@@ -37,22 +37,21 @@ class _NameBodyState extends State<NameBody> {
     super.initState();
   }
 
-  Future<void> _validateName(String name) async {
+  void _validateName() {
     setState(() => _error = name.isEmpty ? Strings.registerNameEmpty : null);
   }
 
   Future<void> _submit(BuildContext context) async {
+    _showError = true;
```

Comment:
Seems odd that we are deciding to show the error before we know whether there was one, or am I missing something?

Reply:
Perhaps bad wording of variable. It means that it should display any error that will get encountered after this point. This is to prevent a validation error from displaying before the user submits.

Reply:
The `error` variable contains the actual error (and is `null` when no error is present)